### DISCUSSION TITLE
Reuse Pages across Spaces

### DISCUSSION THREAD

**Discussion by JosephMorcom** [GitbookIO/community#651] [Feature Requests]
Apologies if this has been requested, I could not find it.

Something I would love to be able to do is create a Page in one Space, then insert that Page into another space, so that if it is updated in one Space, it is updated identically in both Spaces. So, we would be treating the Page as one object which can be used in as many Spaces as required, while only needing to update it in one place. I accept that you will still need to _merge_ all Spaces for the change to actually be visible on all user sides.

The reason is, I have two Spaces that share a lot of content, and currently I make a lot of identical updates to identical Pages (exact same title and content) in both Spaces.

This could potentially be solved with: https://github.com/orgs/GitbookIO/discussions/121#discussioncomment-9444957    But it was not clear if this would allow cross-Space use, and it may not be the best solution for re-using entire Pages.

I know you can link to an external Page within a space, but if the user clicks that link, it opens the Page in the other Space. I would like the user to remain in the same Space, and view it like a normal Page.

If there is already a way to do this, that would be awesome to know.

**Comment by SamyPesse** [comment]
Hi @JosephMorcom, it's not currently possible to do that. But we are working on ways to reuse content in GitBook (as mentioned in https://github.com/orgs/GitbookIO/discussions/121#discussioncomment-9444957), we expect them to be tested and shipped to users over the next few months. It'll work between different spaces.

> and it may not be the best solution for re-using entire Pages

At the moment, there is no plan to support syncing an entire page (title + body), but we'll explore it at some point. A workaround will be to create a page with a single synced block.
